clients/reality-data/src/RealityDataClient.ts

```diff
@@ -32,8 +32,8 @@ export enum RealityDataType {
  * an Azure blob URL, the token may be required to obtain this Azure blob URL or refresh it.
  * The Azure blob URL is considered valid for an hour and is refreshed after 50 minutes.
  * In addition to the reality data properties, and Azure blob URL and internal states, a reality data also contains
-// SWB
- * the identification of the iTwin project to identify the context(used for access permissions resolution) and
+// SWB What do project and context mean here?
```

Comment:
Lets change this entire comment to be,

`the identification of the iTwin to be used for access permissions `.

The rest of these docs need to be completely re-written... They aren't very clear. Let's tackle that in the reality data PR though.

Reply:
Added the new comment.